apps/app/utils/postWishLinkFromShare.ts

```
@@ -1,3 +1,5 @@
+import { postTokenRefresh } from '@/apis/postTokenRefresh';
```

ESLint

[error] Unable to resolve path to module '`@/apis/postTokenRefresh`'.

(import/no-unresolved)